integrations/utils/commonUtils.js

```diff
@@ -116,7 +116,24 @@ function flattenJson(data) {
   return result;
 }
 
+/**
+ * Check whether the passed eventname is mapped in the config
+ * and return the mapped event name.
+ * @param {*} event
+ * @param {*} eventsHashmap
+ * @returns mappedEventName
+ */
+function setEventMappingFromConfig(event, eventsHashmap) {
```

Comment:
Shouldn't it be 'getEventMappingFromConfig'?

Reply:
Changing this to `getEventMappingFromConfig`.